### PR TITLE
fix one namespace issue for chart.

### PR DESCRIPTION
**Pull request by morvencao**
We have exposed a `oneNamespace` parameter in [values.yaml](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L102-L104)
to restrict the applications namespace the controller manages; if not set, controller watches all namespaces, but we haven't get the right parameter in template file.

### CODE REVIEW

**Comment by gyliu513** [comment]
/lgtm cancel

@morvencao please check why lint failed.

```
==> Linting ./install/kubernetes/helm/istio
[ERROR] templates/: render error in "istio/charts/pilot/templates/deployment.yaml": template: istio/charts/pilot/templates/deployment.yaml:33:14: executing "istio/charts/pilot/templates/deployment.yaml" at <.Values.global.istio...>: can't evaluate field oneNameSpace in type interface {}
```

**Comment by morvencao** [comment]
@gyliu513 fixed helm lint issue.

File: install/kubernetes/helm/istio/values.yaml [hunk 1]
```diff
@@ -99,6 +99,10 @@ global:
     s390x: 2
     ppc64le: 2
 
+  # whether to restrict the applications namespace the controller manages
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/values.yaml]: nit: s/whether/Whether

Comment (morvencao) [reply install/kubernetes/helm/istio/values.yaml]: done

**Pull request merged**